addon/-private/system/relationships/state/has-many.js

```diff
@@ -111,6 +111,26 @@ export default class ManyRelationship extends Relationship {
     super.removeCanonicalInternalModelFromOwn(internalModel, idx);
   }
 
+  removeDeletedInternalModelFromOwn(internalModel) {
```

Comment:
i believe the divergence is: https://github.com/emberjs/data/pull/5058/files#r127379185